--- bin/difftool/git-difftool~	2009-05-11 05:05:57.000000000 +0200
+++ bin/difftool/git-difftool	2009-05-31 11:40:47.304899717 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl
+#!/usr/bin/perl
 # Copyright (c) 2009 David Aguilar
 #
 # This is a wrapper around the GIT_EXTERNAL_DIFF-compatible
--- bin/git-cola~	2009-05-11 05:05:57.000000000 +0200
+++ bin/git-cola	2009-05-31 11:40:12.081294820 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
 # -*- python-mode -*-
 """Takes care of starting the main function
 """
